dvc/tree/hdfs.py

```diff
@@ -162,7 +163,7 @@ def copy(self, from_info, to_info, **_kwargs):
                     with closing(
                         hdfs.open_output_stream(tmp_info.path)
                     ) as tmp_fobj:
-                        tmp_fobj.write(from_fobj.read())
```

Comment:
No pbars in `copy()` methods (--external scenario), but that's a common issue in all clouds, so not touching it here yet.